docs/studios/unity/tutorials/github_actions.md

```diff
@@ -0,0 +1,309 @@
+which has filter options that can be passed within `customParameters` within the `Run tests` step above.
+
+:::
+
+## Configure GitHub Actions
+
+The final piece on the GitHub actions side is to configure the secrets that are used in the workflow file above. Visit the secrets
+page for your project by going to your **Repo** > **Settings** > **Secrets and variables** > **Actions**. Add the following secrets
+by clicking the green **New repository secret** button.
+
+1. `RG_HOST` - `https://play.regression.gg`
+2. `RG_API_KEY` - Your API key from Regression Games, which can be found in your [account settings](https://play.regression.gg/account).
+                  Full instructions for this can be found in the [Authentication docs.](studios/api_reference/authentication).
+3. `RG_BOT` - The ID of the bot you want to run. Within the [Bot Manager page](https://play.regression.gg/bots), you can see the ID
+              for a bot in its details below the name of the bot.
+
+![Bot ID](./github_actions_images/bot_id.png)
+
+Once you are done adding these secrets, your secrets section should look similar to the following:
+
+![Secrets for GitHub Actions](./github_actions_images/secrets.png)
+
+## Add a Test to Your Unity Project
+
+The QA Bots are started via a the [Unity Test Runner as a Play Mode test](https://docs.unity3d.com/2019.1/Documentation/Manual/PlaymodeTestFramework.html). 
+In this testing mode, the game starts as if you are playing it, and then terminates once the function finishes.
+
+Complete the following steps to setup the required test - feel free to skip some of these steps if you have existing Play tests
+in your project.
+
+#### 1. Add your scenes to your build
+
+Before adding the test, make sure your desired test scenes are placed inside your build. You can add your scenes to the build
+using the menu option **File** > **Build Settings**.
+
+![Build settings](./github_actions_images/build.png)
+
+#### 2. Configure Test Runner for your project
+
+Next, we need to configure the Unity Test Runner for your project. To do this, go to **Window** > **General** > **Test Runner**.
+Then, in the pane that open, click **Play Mode** > **Create PlayMode Test Assembly Folder**. Create that tests folder with
+the name `Tests`, and then open that folder in your editor. You should see a file called `Tests.asmdef` - open that file and 
+replace it with the contents below, which will include Regression Games as a dependency:
+
+```json
+{
+    "name": "Tests",
+    "references": [
+        "RegressionGames"
+    ],
+    "optionalUnityReferences": [
+        "TestAssemblies"
+    ]
+}
+```
+
+You can learn more about the Test Runner and setup instructions [here](https://docs.unity.cn/560/Documentation/Manual/testing-editortestsrunner.html).
+
+#### 3. Create an `RGBotTests` file to run your bot
+
+Within the `Tests` folder, create a new file called `RGBotTests.cs` and copy the following contents into the file.
```

Comment:
again, we should give them the option to download the file as an alternative to cut/paste contents

Reply:
Is this something we could include in our Unity Package? Maybe they'd just need to set set some variables to configure it for their project but they wouldn't need to go download or do any cut/paste?